apps/client/src/app/app.component.html

```diff
@@ -202,6 +202,7 @@
                 *ngIf="!overlay"
                 [(nzCollapsed)]="settings.compactSidebar"
                 [nzCollapsedWidth]="80 | ifMobile: 0"
+                [nzWidth]="255"
```

Comment:
```suggestion
                [nzWidth]="255 | ifMobile: 200"
```

I think it's better to keep it 200 on mobile, the ifMobile pipe being good for this kind of use case :)

Reply:
Fair enough, I had that before but since I added the z-index I thought it could be wider :)